extensions/parquet/compression/src/main/java/io/deephaven/parquet/compress/DeephavenCompressorAdapterFactory.java

```diff
@@ -140,6 +140,7 @@ public InputStream decompress(final InputStream inputStream, final int compresse
                     decompressor.reset();
                 } else {
                     decompressor = CodecPool.getDecompressor(compressionCodec);
+                    // Not resetting decompressor here since either a new instance or is reset when returned to the pool
```

Comment:
```suggestion
                    // Not resetting decompressor here since it's either a new instance or was reset when previously returned to the pool
```